modules/core/server/controllers/core.dbmodel.controller.js

```diff
@@ -1021,7 +1025,8 @@ _.extend (DBModel.prototype, {
 	},
 
 	paginate: function(query, filter, skip, limit, fields, population, sortby, userCan) {
-		// console.log ('paginate(query=', query, ', filter=', filter, ', skip=', skip, ', limit=', limit, ', fields=', fields, ', population=', population, ', sortby=', sortby, ', userCan=', userCan, ')');
+		//console.log ('paginate(query=', query, ', filter=', filter, ', skip=', skip, ', limit=', limit, ', fields=', fields, ', population=', population, ', sortby=', sortby, ', userCan=', userCan, ')');
+		var debug = false;
```

Comment:
Since this is in the server section of code: process.env.ENV_VAR_HERE.  If wanting constants for the client-side code, see gruntfile.js where this gets built on the fly to public/dist/conf.js.  Comment out is fine.